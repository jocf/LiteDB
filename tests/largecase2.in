set a
set b
set c
list keys
list entries
snapshot
set d
snapshot
del d
snapshot
set a 1
pick a 1
pluck a 1
get a
set a 1 2
pop a
get a
list snapshots
purge a
checkout 1
list keys
rollback 1
list snapshots
bye
