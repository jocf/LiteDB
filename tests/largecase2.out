> ok

> ok

> ok

> c
b
a

> c []
b []
a []

> saved as snapshot 1

> ok

> saved as snapshot 2

> ok

> saved as snapshot 3

> ok

> 1

> 1

> []

> ok

> 1

> [2]

> 3
2
1

> ok

> ok

> c
b

> ok

> 1

> bye
